db/blob/blob_file_builder.cc

```diff
@@ -106,10 +110,10 @@ Status BlobFileBuilder::Add(const Slice& key, const Slice& value,
   }
 
   Slice blob = value;
-  std::string compressed_blob;
+  Slice compressed_blob;
 
   {
-    const Status s = CompressBlobIfNeeded(&blob, &compressed_blob);
+    const Status s = CompressBlobIfNeeded(blob, &compressed_blob);
```

Comment:
`compressed_blob` is not used afterwards. This is ok in the original code because `blob` would point to compressed_blob's data.

Reply:
Yeah, that's true. I just think that we need to put the blob into blob cache and compressed_blob into the secondary cache.  But, we can ignore that part now.